minidump-unwind/src/amd64.rs

```diff
@@ -77,25 +77,14 @@ fn callee_forwarded_regs(valid: &MinidumpContextValidity) -> HashSet<&'static st
     }
 }
 
-fn get_caller_by_frame_pointer<P>(
+async fn get_caller_by_frame_pointer<P>(
```

Comment:
Why is this function `async` now? Just because it may do nontrivial computations?